go.mod

```diff
@@ -4,6 +4,7 @@ go 1.20
 
 require (
 	code.gitea.io/sdk/gitea v0.15.1
+	dagger.io/dagger v0.7.2
```

Comment:
can we make `./ci` its own module? so whoever `go install` goreleaser don't need to download dagger and all this other deps too...

Reply:
Yeah good call